### DISCUSSION TITLE
Firebase storage is paid now!

### DISCUSSION THREAD

**Discussion by TheRealAshik** [Sketchware-Pro/Sketchware-Pro#1490] [Q&A]
### What's the problem?
**Firebase storage** is now paid, Older users who created their project before 25th October can use it for free till Oct 2025, but newer projects are now required to pay for firebase storage.

### What can I do?
Guys! Any **solution**?

**Comment by laymintu** [comment]
Just paay it, 
Have you ever tried your app as a free version dude! 😎

**Comment by H-uey** [comment]
> ### What's the problem?
> **Firebase storage** is now paid, Older users who created their project before 25th October can use it for free till Oct 2025, but newer projects are now required to pay for firebase storage.
> 
> ### What can I do?
> Guys! Any **solution**?

Supabase, not yet tried but I think it's a good alternative.
